<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>QHostInfo</string>
    </property>
    <widget class="QPushButton" name="btn_localHostName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>获取本机主机名(localHostName())</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fromNameArgQString">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>返回指定主机名的IP地址(fromName())</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkOnlyIPv4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>只显示IPV4协议地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_lookupHost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>根据主机名，域名或IP地址查找主机信息(lookupHost())</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editHost">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>QNetworkInterface</string>
    </property>
    <widget class="QPushButton" name="btnALLInterface">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>获取应用程序所在主机的所有网络接口(allInterfaces())</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_allAddress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>521</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>获取应用程序所在主机的所有网络接口,不需要子网掩码和广播(allAddresses())</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>651</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="Network">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
